backend/plugin/db/driver.go

```diff
@@ -158,11 +158,14 @@ type MigrationInfo struct {
 
 // ConnectionConfig is the configuration for connections.
 type ConnectionConfig struct {
-	Host     string
-	Port     string
-	Username string
-	Password string
-	Database string
+	Host string
+	Port string
+	// More hosts and ports are required by elasticsearch.
+	MultiHosts []string
```

Comment:
Can we introduce a new message type called Address? Also use either Host+Port or MultiAddresses, and update comment about it.

type Address struct {
  Host string
  Port string
}

Reply:
yeah, and that sounds much more logical. New .proto files are on their way.

Reply:
yeach, and that sounds way more logical. New .proto files are on their way.